### PR TITLE
chore(v2): update security policy

### PR DESCRIPTION
**Pull request by jstcl**
this pull request is change and updating security polices.

### CODE REVIEW

**Comment by azukashi** [comment]
ngga sekalian fork? ngapain pull request kalo udah ada write permission? hamdeh

File: SECURITY.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ Security Policy for [Exxyll Discord Bot](https://top.gg/bot/848232775798226996)
...
 ## Supported Versions
 
-| Version | Supported                            |
-| ------- | ------------------------------------ |
-| 1.0.x   | :white_check_mark: (deprecated soon) |
-| 2.0.x   | :white_check_mark: (unstable)        |
+| Version | Supported                       |
+| ------- | ------------------------------- |
+| 1.0.x   | :white_check_mark: (deprecated) |
+| 2.0.x   | :white_check_mark:              |
```
Comment (azukashi) [inline SECURITY.md:10]: seriusan nih

**Review by azukashi — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (azukashi) [reply SECURITY.md:10]: yaudah deh

**Review by azukashi — APPROVED** [review]

**Comment by jstcl** [comment]
> ngga sekalian fork? ngapain pull request kalo udah ada write permission? hamdeh

@gifaldyazkaa hehehe, yaudah deh,,, di-fork dulu :sweat_smile:

**Comment by azukashi** [comment]
hadeh kelakuan, serah ah, kalo mau kontrib, kirim aja lgsng.

**Pull request merged**